RLBotPack/Rocketnoodles/src/scenario/__init__.py

```diff
@@ -0,0 +1,10 @@
+from scenario.default_scenario import DefaultScenario
+from scenario.shoot_ball_at_goal import ShootBallAtGoal
+from scenario.dribble_easy import DribbleEasy
+from scenario.dribble_hard import DribbleHard
+from scenario.intercept import Intercept
+from scenario.side_shot import SideShot
+from scenario.ball_through_center import BallThroughCenter
+
+__all__ = ["DefaultScenario", "ShootBallAtGoal", "DribbleEasy", "DribbleHard", "SideShot", "Intercept",
+           "BallThroughCenter"]
```

Comment:
You could omit your tests also, but up to you.

Reply:
We would like to leave these in. They will probably play a bigger role in the future!

Reply:
OK!